swap_meet/vendor.py

```diff
@@ -1,2 +1,90 @@
+# from swap_meet.item import Item
+
+
 class Vendor:
-    pass
\ No newline at end of file
+    def __init__(self, inventory=None):
+        self.inventory = [] if inventory is None else inventory
+
+    def add(self, item):
+        self.inventory.append(item)
+        return item
+
+    def remove(self, item):
+        if item in self.inventory:
+            self.inventory.remove(item)
+            return item
+        else:
+            return False
+
+    def get_by_id(self, item_id):
+        self.item_id = item_id
```

Comment:
Careful here, because you are creating a new `Vendor` attribute called `item_id` when the `Vendor` class does not have an attribute like that in its constructor. Consider removing this line altogether.